src/Microsoft.ML.FastTree/FastTreeArguments.cs

```diff
@@ -174,16 +174,16 @@ public abstract class TreeOptions : TrainerInputBaseWithGroupId
         /// <summary>
         /// The seed of the random number generator.
         /// </summary>
-        [Argument(ArgumentType.LastOccurenceWins, HelpText = "The seed of the random number generator", ShortName = "r1")]
-        public int RngSeed = 123;
+        [Argument(ArgumentType.LastOccurenceWins, HelpText = "The seed of the random number generator", ShortName = "r1,RngSeed")]
+        public int Seed = 123;
 
         // this random seed is only for active feature selection
         /// <summary>
         /// The seed of the active feature selection.
         /// </summary>
-        [Argument(ArgumentType.LastOccurenceWins, HelpText = "The seed of the active feature selection", ShortName = "r3", Hide = true)]
+        [Argument(ArgumentType.LastOccurenceWins, HelpText = "The seed of the active feature selection", ShortName = "r3,FeatureSelectSeed", Hide = true)]
```

Comment:
>FeatureSelectSeed [](start = 121, length = 17)

I notice that we're doing this -- is the intent to maintain back-compat with existing entry-points? If it is so, I *might* prefer to do `Name` attribute, so we suppress the new name. And if we don't *care* about back compat, then I think we can just say, "you know what, this is the new name now." I'd rather just do the "hard break" now and drop these existing things. Do we have a solid scenario that requires this back-compat on the arguments themselves?

Reply:
So my preference would be, let's not do this.

---
In reply to: [261735355](https://github.com/dotnet/machinelearning/pull/2753#discussion_r261735355) [](ancestors = 261735355)

Reply:
Is this comment applied to every other argument in other trainers?

---
In reply to: [261735502](https://github.com/dotnet/machinelearning/pull/2753#discussion_r261735502) [](ancestors = 261735502,261735355)